application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -4618,6 +4666,50 @@ input TransitPreferencesInput {
   can be found under the street mode preferences.
   """
   board: BoardPreferencesInput
+  """
+  A list of filters for which trips should be included or excluded. A trip will be included if it matches 
+  with at least one filter.
```

Comment:
If we have one "filter" which defines an include that includes a trip, but also another that defines an exclude. Then I guess it's not enough that the trip matches the include filter?

Reply:
Or maybe this is correct, otherwise it wouldn't be possible to include only route from an agency? So for example:
```
  filters: [
    {
      include: {
        routes: ["F:route1"]
      }
    },
    { 
      exclude: {
        agencies: ["F:agency1"]
      }
    }
  }]
  ```